test_runner/regress/test_bad_connection.py

```diff
@@ -15,7 +15,13 @@ def test_compute_pageserver_connection_stress(neon_env_builder: NeonEnvBuilder):
     pageserver_http.configure_failpoints(("simulated-bad-compute-connection", "50%return(15)"))
 
     env.neon_cli.create_branch("test_compute_pageserver_connection_stress")
-    endpoint = env.endpoints.create_start("test_compute_pageserver_connection_stress")
+    try:
```

Comment:
Hmm, isn't this test meant to validate that we start operate cleanly even when we have connection failures?  It seems strange to explictly work around failures to start up.